src/loading/shaderpack/render_graph_builder.cpp

```diff
@@ -1,11 +1,9 @@
-/*!
- * \author ddubois
- * \date 17-Sep-18.
- */
-
 #include "render_graph_builder.hpp"
```

Comment:
Why didn't this get sorted to the bottom of the includes?

Reply:
Because its the header belonging to the source file